### PR TITLE
use sha256

### PR DESCRIPTION
**Pull request by Nesurion**
Use sha256 hash instead of sha1 for the current release.
Future releases will use the sha256 hash automatically, as soon as  https://github.com/giantswarm/cli/pull/559 is merged.

### CODE REVIEW

**Comment by Nesurion** [comment]
@giantswarm/cena RFR

File: Formula/swarm-client.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 class SwarmClient < Formula
   homepage "https://giantswarm.io"
-  # openssl dgst -sha1 <file>
+  # openssl dgst -sha256 <file>
   url "https://downloads.giantswarm.io/swarm/clients/0.27.0/swarm-0.27.0-darwin-amd64.tar.gz"
-  sha1 "7432927c2609cc9edeac912123b468f9d055cbf1"
-  version "0.27.0"
+  sha256 "3defc4bac2f7e64b299b87a7a7199e855f65ce0f1e2943f390e047310ade9c47"
+  version "0.26.0"
```
Comment (JosephSalisbury) [inline Formula/swarm-client.rb]: eh? shouldn't this be 0.27.0?

File: Formula/swarm-client.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 class SwarmClient < Formula
   homepage "https://giantswarm.io"
-  # openssl dgst -sha1 <file>
+  # openssl dgst -sha256 <file>
   url "https://downloads.giantswarm.io/swarm/clients/0.27.0/swarm-0.27.0-darwin-amd64.tar.gz"
-  sha1 "7432927c2609cc9edeac912123b468f9d055cbf1"
-  version "0.27.0"
+  sha256 "3defc4bac2f7e64b299b87a7a7199e855f65ce0f1e2943f390e047310ade9c47"
```
Comment (xh3b4sd) [inline Formula/swarm-client.rb]: Just for my interest because I have no idea. Where is this hash coming from?

Comment (JosephSalisbury) [inline Formula/swarm-client.rb]: it is the sha256 of the downloaded tarball

File: Formula/swarm-client.rb (hunk 1 above)
Comment (Nesurion) [inline Formula/swarm-client.rb]: jep...
fixed

**Comment by JosephSalisbury** [comment]
LGTM

**Comment by zeisss** [comment]
@Nesurion can you do the same in https://github.com/giantswarm/homebrew-giantswarm ?

**Pull request merged**